api/models/model.py

```diff
@@ -1045,7 +1045,7 @@ def message_files(self):
             files.append(file)
 
         result = [
-            {"belongs_to": message_file.belongs_to, **file.to_dict()}
+            {"belongs_to": message_file.belongs_to, "upload_file_id": file.related_id, **file.to_dict()}
```

Comment:
Please don't include this field in the returned result, to fix the 'more like this' function, you can query MessageFile manually and build the File object by using `file_factory.build_from_message_files` in https://github.com/langgenius/dify/blob/c135967e591488a325b868dd20c2283fb4023d50/api/core/app/apps/completion/app_generator.py#L227